tssc/step_implementers/unit_test/junit.py

```diff
@@ -7,42 +7,53 @@
 Could come from either configuration file or
 from runtime configuration.
 
-| Configuration Key | Required? | Default | Description
-|-------------------|-----------|---------|-----------
-| `TODO`            | True      |         |
+| Configuration Key | Required? | Default     | Description
+|-------------------|-----------|-------------|-----------
+| `pom-file`        | True      | `'pom.xml'` | pom used to check for reportsDirectory definition
```

Comment:
The description doesn't seem quite right

Reply:
Updated.